organization/src/test/scala/com/improving/app/organization/domain/OrganizationSpec.scala

```diff
@@ -57,7 +58,11 @@ class OrganizationSpec
     assert(response.isSuccess)
   }
 
-  private def suspendOrganization(organizationId: String, p: ActorRef[OrganizationRequestEnvelope], probe: TestProbe[StatusReply[OrganizationEvent]]) = {
+  private def suspendOrganization(
+      organizationId: String,
+      p: ActorRef[OrganizationRequestEnvelope],
+      probe: TestProbe[StatusReply[OrganizationEvent]]
+  ) = {
```

Comment:
This kind of (automatic, I'm assuming) style change in an area you didn't touch has a tendency to cause merge conflicts

Reply:
Yes, but I am only using the scalafmt configuration that is included in the project. I have a feeling code is only being refactored like this because other developers have IntelliJ configurations that don't use the scalafmt configuration provided. Otherwise I am not sure why there is a discrepancy between my automatic formatting and others'

Reply:
In the scalafmt docs they use the `.` prefix notation for their files. https://scalameta.org/scalafmt/docs/configuration.html

in mac this just so happens to make them hidden files
https://stackoverflow.com/questions/53579232/cannot-view-gitignore-file-in-mac

not sure whether to take away the `.` or leave as is

Reply:
that's the behavior on linux as well. It seems to me that IntelliJ should just be picking up on the presence of the file and configuring accordingly, but it seems its formatting configuration is also global as opposed to per-project, which also seems ridiculous to me. I wonder if there's an IntelliJ feature request for this.

Another thing that might be a good idea (out of scope for this PR of course) is to add a format check to ci.

Reply:
Good idea on the last one - will make a ticket